packages/ember-mirage/addon/setup-mirage.js

```diff
@@ -1,17 +1,32 @@
 import { settled } from '@ember/test-helpers';
 
-import startMirage from './start-mirage';
+import { startMirage } from './start-mirage';
```

Comment:
startMirage is an export default, does this work?

Reply:
I think non embroider modules are trolling me. It should not work

Reply:
I thought about this last night . I think it does because you exported it as default, but also named it.